server.js

```diff
@@ -37,32 +19,17 @@ var titterSender = require('./titter-sender');
 app.set('views', __dirname + '/views');
 app.set('view engine', 'ejs');
 
-var SessionStore = require('express-mysql-session');
 var cookie_secret = nconf.get("server:cookiesecret");
 app.use(cookieParser(cookie_secret));
-var session_options = {
-		host: MYSQL_HOST,
-		port: 3306,
-		user: MYSQL_USER,
-		password: MYSQL_PASSWORD,
-		database: MYSQL_DB
-}
-var sessionStore = new SessionStore(session_options);
-app.use(session(
-	{
-
-		secret: cookie_secret,
-		saveUninitialized: true,
-		store: sessionStore,
-		resave: true,
-		cookie: {
-			secure:false,
-			domain:DOMAIN,
-			maxAge: 1000 * 60 * 60 * 24 * 30
-		},
-		key: 'sid'
-	}
-));
+app.use(session({
+	cookie: {
+		maxAge: 36000000,
+		httpOnly: false
+	},
+	secret: cookie_secret,
+	saveUninitialized: true,
+	resave: false
+}));
```

Comment:
All wrio applications sharing single session, saved in sessions store. this change would break titter operation. When moving to mongo, mysql session store should be synchronously changed to mongostore for all wrio application.